src/parser/core/Events.js

```diff
@@ -90,6 +90,13 @@ const Events = {
    * BEWARE: Blizzard also sometimes uses this event type for mechanics and spell ticks or bolts. This can even occur in between a begincast and cast finish!
    * @returns {EventFilter}
    */
+  get beacontransfer() {
+    return new EventFilter('beacontransfer');
+  },
+  /**
```

Comment:
This jsdoc should be *before* the `get beacontransfer() {`, and the one above `beacontransfer` should be here